src/native/libs/System.Globalization.Native/pal_casing.m

```diff
@@ -6,13 +6,13 @@
 #include "pal_errors.h"
 
 #import <Foundation/Foundation.h>
+#include <unicode/uchar.h>
```

Comment:
this should already be included by pal_icushim_internal.h

Reply:
It is not included by `pal_icushim_internal.h`. In  `pal_icushim_internal.h` are included our ICU headers for OSX platform but not for `TARGET_MACCATALYST`/`TARGET_IOS`/`TARGET_TVOS`.
Now I include from system ICU only the ones that will be needed in `.c` file.

Reply:
I will move all these includes to `pal_icushim_internal.h` to make it more clear.